spec/models/users/notification_setting_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe Users::NotificationSetting, type: :model do
+  let!(:user) { create(:user) }
+  let(:notification_setting) { described_class.find_by(user_id: user.id) }
+
+  describe "validations" do
```

Comment:
I'm tagging @citizen428 and @rhymes to get their take on this, @takmar, but I **believe we can remove this entire `describe` block**, and the corresponding solitary validation in app/models/users/notification_setting.rb:7 (line 7).

When running the spec, I get the following warning from `shoulda-matchers` gem:
```
Warning from shoulda-matchers:

You are using `validate_inclusion_of` to assert that a boolean column
allows boolean values and disallows non-boolean ones. Be aware that it
is not possible to fully test this, as boolean columns will
automatically convert non-boolean values to boolean ones. Hence, you
should consider removing this test.
```

Reply:
Alas, this column doesn't have a non-null constraint so we would be able to accidentally unset this without the validation. Since this is tangential to this PR I think we should leave it in and clean up all the boolean validations at the same time, adding constraints where necessary.

Reply:
Yes, we can remove this, see http://www.chrisrolle.com/en/blog/boolean-attribute-validation.

Reply:
I removed the code related to validation. Thank you!
https://github.com/forem/forem/pull/14226/commits/93095a325e30e29fdd1f8369644711f956a096c0